dp_wizard/app/__init__.py

```diff
@@ -31,13 +34,76 @@ def ctrl_c_reminder():  # pragma: no cover
     print("Session ended (Press CTRL+C to quit)")
 
 
+def _make_demo_csv(path: Path, contributions):
```

Comment:
This function is just being moved down, closer to where it is needed, and the test is brought in as well. (This is a little big for a doctest, but if we need to tweak the columns, it will nice not to need to switch between files.)